src/apps/clients/permissions.clj

```diff
@@ -31,6 +31,11 @@
   [perms]
   (into {} (map (juxt (comp uuidify :name :resource) :permission_level) perms)))
 
+(defn- group-abbreviated-permissions
+  "Groups abbreviated permissions by resource ID. The resource ID must be a UUID."
+  [perms]
+  (into {} (map (juxt (comp uuidify :resource_name) :permission_level) perms)))
+
```

Comment:
The changes in this file are all related to the new endpoint in the `permissions` service.